packages/dds/map/src/directory.ts

```diff
@@ -1305,13 +1305,8 @@ class SubDirectory extends TypedEventEmitter<IDirectoryEvents> implements IDirec
 			throw new Error("Undefined and null keys are not supported");
 		}
 
-		// Create a local value and serialize it.
+		// Create a local value.
 		const localValue = this.directory.localValueMaker.fromInMemory(value);
-		const serializableValue = makeSerializable(
```

Comment:
@markfields I actually don't think this can be removed - without this, the visibility of the datastores is not getting set correctly. This also predated #19242, so I would think it's ok to leave.

Reply:
As discussed - sounds like a bug.  We'll follow up separately.